util/util.go

```diff
@@ -29,13 +29,21 @@ func init() {
 	}
 }
 
+func toSlash(v string) string {
+	v = strings.Replace(v, "\\", "/", -1)
+	if os.PathSeparator != '\\' && os.PathSeparator != '/' {
+		v = strings.Replace(v, string(os.PathSeparator), "/", -1)
+	}
```

Comment:
Is there an operating system with the path separator other than `/` or `\` that Go supports? Or that's in wide use? If so that's a separate issue because other things will fail.